src/Config.php

```diff
@@ -165,13 +172,35 @@ protected function configureProcessors(array $processors)
     /**
      * Configure the loggers
      *
-     * @param  array $loggers Array of logger options
+     * @param array $loggers Array of logger options
      */
     protected function configureLoggers(array $loggers)
     {
+
+        $instantiatedLoggers = array();
+
+        if (array_key_exists('default', $loggers)) {
```

Comment:
I would suggest a comment above this to explain why you are doing the `default` logger first, and why you are removing it from the config file. It is fairly obvious to us, but a novice may appreciate the hint.